### PR TITLE
POL-85 Office 365 Security Alerts

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

initial release of Office 365 Security Alerts policy

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: saas/office365/security_alerts/o365-security-alerts.pt [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    }
+  EOS
+end
+
+datasource "ds_filter_alerts" do
+  run_script $js_filter_alerts,$ds_o365_alerts
+end
+
+script "js_filter_alerts", type: "javascript" do
```
Comment (cdwilhelm) [inline saas/office365/security_alerts/o365-security-alerts.pt]: Add some sorting.  maybe by date?

Comment (dfrankel33) [reply saas/office365/security_alerts/o365-security-alerts.pt]: security alerts should always raise an incident.  filtering is done by criticality

File: saas/office365/security_alerts/o365-security-alerts.pt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+auth "azure_auth", type: "oauth2" do
+  token_url join(["https://login.microsoftonline.com/",$param_azure_tenant_id,"/oauth2/v2.0/token"])
+  grant type: "client_credentials" do
+    client_id cred("O365_AAD_APPLICATION_ID")
+    client_secret cred("O365_AAD_APPLICATION_KEY")
+    additional_params do {
+      "scope" => "https://graph.microsoft.com/.default"
```
Comment (cdwilhelm) [inline saas/office365/security_alerts/o365-security-alerts.pt]: Danger is complaining about this.  the host should be added to the exclude_hosts array in the Dangerfile.  Can yo include that in your PR.

File: saas/office365/security_alerts/o365-security-alerts.pt [hunk 3]
```diff
@@ -0,0 +1,144 @@
+name "Office 365 Security Alerts"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will identify Security Alerts that have been raised in Office 365. Policy Managers can minimize the notifications by choosing to only be alerted by certain severity level(s). See the [README](https://github.com/rightscale/policy_templates/tree/master/fsm/office_365/security_alerts/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline saas/office365/security_alerts/o365-security-alerts.pt]: Danger doesn't like these links.   the github link should be changed to flexera and the docs should be an https.

**Pull request merged**